core/server/utils/url.js

```diff
@@ -349,12 +349,28 @@ function isSSL(urlToParse) {
     return protocol === 'https:';
 }
 
+function redirect301(res, redirectUrl) {
+    res.set({'Cache-Control': 'public, max-age=' + config.get('caching:301:maxAge')});
+    return res.redirect(301, redirectUrl);
+}
+
+function adminRedirect(res, adminPath, permanent) {
```

Comment:
`utils.url.redirectToAdmin`?
I find that more readable + is similar to `redirect301`.

Reply:
furthermore: how about options pattern?
I never know from outside (res, adminPath, **true**) means.

Reply:
Have renamed. Not sure about options pattern... lemme think

Reply:
I prefer either:

`redirectToAdmin301(res, path)` && `redirectToAdmin(res, path)`

orrrr

`redirectToAdmin(301, res, path)` &&  EITHER `redirectToAdmin(res, path)` OR `redirectToAdmin(302, res, path)`

Reason for not using options pattern is there should be no further options & 3 args is not too many IMO 😁

Reply:
Hmm....second suggestion.

Reply:
Made more changes